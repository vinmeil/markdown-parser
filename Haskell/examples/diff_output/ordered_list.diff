diff --git a/examples/expected_output/ordered_list.html b/examples/output/ordered_list.html
index 0c41095..b53e672 100644
--- a/examples/expected_output/ordered_list.html
+++ b/examples/output/ordered_list.html
@@ -1,24 +1,13 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <title>Test</title>
-</head>
-
 <body>
     <ol>
-        <li>Item 1
-            <ol>
-                <li>Sub Item 1</li>
-                <li>Sub Item 2</li>
-                <li>Sub Item 3</li>
-            </ol>
-        </li>
+        <li>Item 1</li>
+        <ol>
+            <li>Sub Item 1</li>
+            <li>Sub Item 2</li>
+            <li>Sub Item 3</li>
+        </ol>
         <li><strong>Bolded Item 2</strong></li>
         <li>Item 3</li>
         <li>Item 4</li>
     </ol>
 </body>
-
-</html>
