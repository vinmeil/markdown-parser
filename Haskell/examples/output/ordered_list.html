<body>
    <ol>
        <li>Item 1</li>
        <ol>
            <li>Sub Item 1</li>
            <li>Sub Item 2</li>
            <li>Sub Item 3</li>
        </ol>
        <li><strong>Bolded Item 2</strong></li>
        <li>Item 3</li>
        <li>Item 4</li>
    </ol>
</body>
