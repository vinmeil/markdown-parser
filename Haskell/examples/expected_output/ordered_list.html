<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Test</title>
</head>

<body>
    <ol>
        <li>Item 1
            <ol>
                <li>Sub Item 1</li>
                <li>Sub Item 2</li>
                <li>Sub Item 3</li>
            </ol>
        </li>
        <li><strong>Bolded Item 2</strong></li>
        <li>Item 3</li>
        <li>Item 4</li>
    </ol>
</body>

</html>
